specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/HttpRouteConfig.json

```diff
@@ -0,0 +1,619 @@
+            "schema": {
+              "$ref": "#/definitions/HttpRouteConfig"
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create or Update Http Route": {
+            "$ref": "./examples/HttpRouteConfig_CreateOrUpdate.json"
+          },
+          "Create or Update Http Route Prefix Rule": {
+            "$ref": "./examples/HttpRouteConfig_CreateOrUpdatePrefix.json"
+          },
+          "Create or Update Http Route Path Separated Prefix Rule": {
+            "$ref": "./examples/HttpRouteConfig_CreateOrUpdatePathSepPrefix.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "HttpRouteConfig"
+        ],
+        "summary": "Update tags of a manged http route object",
+        "description": "Patches an http route config resource. Only patching of tags is supported",
+        "operationId": "HttpRouteConfig_Update",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "description": "Name of the Managed Environment.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "httpRouteName",
+            "in": "path",
+            "description": "Name of the Http Route Config Resource.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "httpRouteConfigEnvelope",
+            "in": "body",
+            "required": true,
+            "description": "Properties of http route config that need to be updated",
+            "schema": {
+              "$ref": "#/definitions/HttpRouteConfigPatch"
+            }
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Managed Http Routes updated successfully.",
+            "schema": {
+              "$ref": "#/definitions/HttpRouteConfig"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Patch Managed Http Route": {
+            "$ref": "./examples/HttpRouteConfig_Patch.json"
+          }
+        }
+      },
+      "delete": {
```

Comment:
is it a long running operation?

Reply:
no, will send fix

Reply:
Confirm with Chris if resource creation happens async and will make this async. 
Also to handle transient issues with managed cluster incase api-server availablity/throttling

Reply:
we will keep this as non long running
cc: @Tratcher